.gitignore

```diff
@@ -4,6 +4,7 @@
 .env
 .DS_Store
 venv/
+.venv/
```

Comment:
The `.venv/` line is correctly included to ignore virtual environment directories. Ensure that this matches the actual directory name used in your project. If your virtual environment directory is named differently, update this line accordingly.